snapcraft/cli/_errors.py

```diff
@@ -230,6 +232,10 @@ def validate(value):
 
 
 def _submit_trace(exc_info):
+    kwargs = dict()  # Dict[str,str]
+    if "+git" not in snapcraft.__version__:
+        kwargs["release"] = snapcraft.__version__
```

Comment:
I have a question though: this means that folks on edge (etc.) will be reporting errors, but the report won't have a `release` associated with it. How does sentry treat those reports compared to reports coming from stable, where `release`s are reported?